src/code-templates/.prettierrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   trailingComma: "es5",
   tabWidth: 2,
-  semi: false,
+  semi: true,
```

Comment:
@DanielGluskin @rluvaton The main idea of Prettier is "we set good default so you don't deal with it"

Should we really express an opinion here?

Reply:
1. yes in my opinion. Will correct.
2. It's a philosophic question. On one hand, we should try stick to the defaults. On the other, if we have the option to use a settings file, we should use it time-to-time. So the I think it's fine to edit some rules, if we mostly agree on them, like in this case.